app/src/main/res/layout/fragment_profile_my_events.xml

```diff
@@ -89,7 +89,13 @@
                 android:visibility="visible" />
 
         </LinearLayout>
-
+            <LinearLayout
+                android:id="@+id/llStatus"
```

Comment:
The `llStatus` layout is repeated in each of these 4 layout files. Can you refactor this into a standalone layout file and use the `<include` attribute to import the standalone layout